lib/tasks/database_cleanup.rake

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+namespace :database do
```

Comment:
Shouldn't we use the `db` namespace? to be consistent with all other DB tasks

Reply:
I think `database_cleanup` was actually too vague. I moved the task under `multitenant:tenants:cleanup_orphans`, I think it belongs there.